javascript/packages/orchestrator/src/orchestrator.ts

```diff
@@ -161,6 +162,9 @@ export async function start(
     if (networkSpec.settings.node_spawn_timeout)
       client.timeout = networkSpec.settings.node_spawn_timeout;
     network = new Network(client, namespace, tmpDir.path);
+    if (options?.setGlobalNetwork) {
+      options.setGlobalNetwork(network);
+    }
```

Comment:
I think you can just do
```suggestion
    options?.setGlobalNetwork(network);
```

Reply:
Just tried and Typescript gives me:

```Cannot invoke an object which is possibly 'undefined'.```

Reply:
Got ```Property 'b' does not exist on type 'never'.``` if I try to copy the code in the project to use our TS config